paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass.cc

```diff
@@ -354,7 +354,9 @@ void CPUQuantizeSquashPass::OpDequantSquash(Graph* graph) const {
           FindOutputNameByVarName(any_op->Op(), dequant_in->Name());
 
       if (output_name.empty()) return;
-
+      if (any_op->Op()->Type() == "conv2d") {
+        any_op->Op()->SetType("fused_conv2d");
```

Comment:
So all int8-oneDNN kernels should be executed as fused kernels by default?

Reply:
Yes, we are trying to delete the extra inputs and attributes in base op, so some extra attributes for int8-oneDNN kernel are also removed, currently we have to put them into fused kernel to execute because no better choice. 
I think a good way to execute int8-oneDNN kernel is creating a new kernel for int8-oneDNN, but it is difficult to implement at the current stage. Maybe we could come up with a good solution in the future.

Reply:
Ok, thank you for explaining